example/sankey-diagram.md

```diff
@@ -0,0 +1,29 @@
+```js
```

Comment:
examples need a title, as defined by the very first `#` present. Please add a title and an intro paragraph, like in the Network example: https://raw.githubusercontent.com/d3plus/d3plus-network/master/example/getting-started.md